patterns/2-structured/trusted-committer.md

```diff
@@ -6,51 +6,49 @@ Trusted Committer
+In these situations, project maintainers seek ways to recognize and reward the
 work of the contributor above and beyond single contributions.
 
 ## Problem
 
 - Project maintainers want to find ways to scale their ability to support a project
 - Project maintainers want to find ways to lengthen the value delivered by a project
 - Project maintainers want to visibly reward frequent contributors and empower them to amplify their value contribution.
-- Lack of language for recognizing contributions across teams within an organization
+- Lack of mechanism for recognizing contributions across teams within an organization
 
 ## Context
 
 - You are the maintainer of a cross-team library, service, or shared resource
 - You receive regular contributions
+- You receive regular feature requests
+- You receive regular bug-fix requests
 - There are motivated contributors looking to build expertise through InnerSource projects
 
 ## Forces
 
-- Over the lifecycle of a project the focus of the maintainers may shift away to accommodate changing business priorities
-- Contributors seek visible artifacts of their contributions demonstrating value
+- Over the lifecycle of a project, the focus of the maintainers may shift away to accommodate changing business priorities
+- Contributors seek visible recognition of their contributions, demonstrating value
 - Maintaining a project of reasonable complexity is taxing for a small team
+- Developing project features at scale is taxing for a small team
 
 ## Solution
 
 ### Defining the Trusted Committer Role for a Project
 
-What a Trusted Committer handles is up to each project and its maintainers.
-Whatever shape your Trusted Committer role takes, make sure it's clearly
-documented somewhere in your project. This sets expectations for new community
-members and outlines the role for future candidates.
+What a Trusted Committer handles is up to each project and its maintainers. Ensure you document within the project the scope of your Trusted Committer role. Clear documentation sets expectations for new community members and establishes the role for future candidates.
 
-Below we've provided a few guidelines for what Trusted Committers can be
-invited to do:
+The following are a few guidelines for identifying a potential Trusted Committer:
 
-* If a candidate participates often in community channels (e.g. Slack, JIRA issue triaging, etc.) then becoming a Trusted Committer formalizes their role in community support.
-* A good candidate for a Trusted Committer, is someone who frequently submits code, documentation, or other repository changes.  Start by including this person on pull requests. If they are actively engaging in pull requests, consider approaching them about opportunities for further collaboration on the project.
+* An active participant in community channels (e.g., Slack, JIRA issue triaging, etc.) becomes a Trusted Committer, thereby formalizing their role in community support.
```

Comment:
```suggestion
* An active participant in community channels (Slack, JIRA issue triaging, etc.) becomes a Trusted Committer, thereby formalizing their role in community support.
```

I am sure the form "e.g., " is grammatically correct, however I find it hard to read.

As the "e.g." and the "etc" are both saying kind of the same, I propose to remove one of them.

Reply:
Totally agree I am for reducing text. In technical writing it is about being as brief as possible. Im sure there are a lot more areas we can do but this is iterative :) Great team work 👍